vita3k/renderer/src/texture_yuv.cpp

```diff
@@ -9,9 +9,32 @@ extern "C" {
 }
 
 namespace renderer::texture {
+
+static SwsContext *s_render_sws_context{};
+static size_t res[2] = { 0, 0 };
+SwsContext *get_sws_context(size_t width, size_t height) {
+    bool recreate = false;
+    if (res[0] != width || res[1] != height) {
+        recreate = true;
+        res[0] = width;
+        res[1] = width;
```

Comment:
```suggestion
        res[1] = height;
```
I assume `res[0]` is for the width and `res[1]` for the height.

Reply:
Ah yes, it should be. woops.